day3/day3.py

```diff
@@ -0,0 +1,75 @@
+import logging
+import re
+from typing import List, Union, Tuple
+
+import numpy as np
+
+import day2
+
+
+def parse_claims(mylines: Union[List[str], Tuple[str]]):
+    '''
+    Parses the elves claims
+
+    :param mylines: A list object containing the fabric claims as separate entities
+    to be parsed in the format (#int @ int,int: int x int)
+    :return: For each entity in the original list this function converts them to [['str','str','str','str']]
+    '''
+    if not mylines:
+        raise ValueError("mylines can not be empty")
+
+    if type(mylines) not in {tuple, list}:
+        raise TypeError("mylines must be tuple, list, or generator")
+
+    fabric_coordinates = []
+    for i in mylines:
+        # gets rid of the empty spaces
+        j = i.replace(' ', '')
+        parsed_claim = re.split('#|@|,|:|x', j)
+        # remove the #claim
+        fabric_coordinates.append(parsed_claim[1:6])
+    return fabric_coordinates
+
+
+def parse_fabric_list(claims, size):
```

Comment:
Add some type hints here, along the lines of `def functionname(arg1: type1, arg2: type2) -> (returntype1, returntype2):`